script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_2.txt

```diff
@@ -23,7 +23,7 @@ The one that's controlling those zombies or something."
 <Page7>
 "Castle Lords and Ancestors are not necessarily the same.
 The more powerful a Dead Apostle is, the easier it is for them to become a Castle Lord, and a Castle Lord is something more like a local ruler.
-Ancestors are above that... They can produce these Castle Lords, not to mention other, higher vampires."
+Ancestors are something above that... They're a higher class of vampire that can produce these feudal lords."
```

Comment:
Castle Lords, keep the terms constant

Reply:
it's different in jp
「城主と祖は必ずしも同じじゃないわ。
　強力な死徒ほど城主になりやすいだけで、城主はいわば地方領主みたいなものなの。
　祖はその上……領主たちを輩出する、さらに上の吸血鬼」

Reply:
yeah Petrikow asked for this change during the group PR. I had originally left it as the same term because I didn't feel like there was any reason to use the other term, but I kinda like this change so I say it stays.